src/JWT/Jwk/JwtWebKeySet.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+#if NET462_OR_GREATER || NET6_0_OR_GREATER || NETSTANDARD2_0_OR_GREATER
+using System.Text.Json.Serialization;
+#endif
+
+namespace JWT.Jwk
+{
+    /// <summary>
+    /// A JWK Set JSON data structure that represents a set of JSON Web Keys
+    /// specifed by RFC 7517, see https://datatracker.ietf.org/doc/html/rfc7517
+    /// </summary>
+    public sealed class JwtWebKeySet
```

Comment:
Does a class need the default ctor decorated with `[JsonConstructor]` if it has none?

Reply:
I think, it could be removed. I made it with the same rules as in `JwtHeader`:
https://github.com/jwt-dotnet/jwt/blob/7ebbddf34a003f9934b4f4655ade32c29735aa67/src/JWT/Builder/JwtHeader.cs#L15

Maybe there is something, I'm not aware about, could you clarify, why we need it in `JwtHeader`, please?